etc/grafana/dashboards/overview.json

```diff
@@ -2602,19 +2619,21 @@
         "h": 1,
         "w": 24,
         "x": 0,
-        "y": 101
+        "y": 102
       },
-      "id": 89,
+      "id": 79,
       "panels": [],
-      "title": "Transaction Pool",
+      "repeat": "instance",
+      "repeatDirection": "h",
+      "title": "Blockchain Tree ($instance)",
       "type": "row"
     },
     {
       "datasource": {
         "type": "prometheus",
         "uid": "${DS_PROMETHEUS}"
       },
-      "description": "Tracks a heuristic of the memory footprint of the various transaction pool sub-pools",
+      "description": "The block number of the tip of the canonical chain from the blockchain tree.",
```

Comment:
Ok, it's a bit painful to work on this issue because every time there's a commit that modifies `[etc/grafana/dashboards/overview.json](https://github.com/paradigmxyz/reth/pull/3336/files/64e902a073593964ccefe3e7621cb174428bc9da#diff-3694b6d47929cdc8535d9ab68c3ae13b86bfd87529024b4525484b413c351a2c)`, it breaks my PR and the rebase/merge never work unfortunately... I suggest opening a fresh PR with the changes and we try to merge it asap, what do you think?

Reply:
I think continuing with this PR is fine. You can checkout to the latest main, and then re-apply your title, templating and `{instance=~\"$instance\"}` changes.

Reply:
@shekhirin Should be good now!